src/event-emitter.js

```diff
@@ -142,7 +225,10 @@ const eventEmitter = ({ getHandlers, registerEventHandler }) => {
     const emitWaterfall = (event, ...args) => {
         const { handlers, eventObject, meta } = getEventObject(event);
         meta.nextInput = args;
-        for (const handler of handlers) {
+        for (const handler of _.flatten(handlers)) {
+            if (!handler) {
```

Comment:
[Reminder] **No issue**, reminder for later. We discussed attempting to catch the (if (!handler)) instances earlier.